pkg/k8s/watchers/cilium_endpoint.go

```diff
@@ -172,6 +175,17 @@ func (k *K8sWatcher) endpointUpdated(oldEndpoint, endpoint *types.CiliumEndpoint
 		return
 	}
 
+	if option.Config.EnableHighScaleIPcache &&
+		!identity.IsWellKnownIdentity(id) {
```

Comment:
non-blocking: I can't help but wonder if we could solve this particular problem in a more targeted way and either change the set of resources that are watched (ie do the well known identity check server-side), or remove the whole CEP watcher altogether. This is a potentially huge amount of bandwidth, not to mention kube-apiserver resource consumer etc. just to share identities for a handful of IP addresses.

Reply:
Yeah, good point. I do wonder whether we should just do the same thing as "Disable pod watcher" commit in the CEP watcher here.